k8s/ceramic/manifests/network.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Is this file generated using the `kube-rs` tooling?

Reply:
I added this file